manifests/e/Easeware/DriverEasy/5.8.1.41398/Easeware.DriverEasy.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Automatically updated by the winget bot at 2023/Dec/05
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Easeware.DriverEasy
+PackageVersion: 5.8.1.41398

```

Comment:
@stephengillie `PackageVersion: 5.8.1.41398` is not right. It Should be `PackageVersion: 5.8.1` otherwise it will always upgrade!

Reply:
I've recently added automation to detect and not approve this kind of version parameter expansion.